src/main/preload.js

```diff
@@ -42,10 +51,23 @@ window.requireAppModule = function(moduleName, isNear) {
 };
 
 window.requireNodeModule = function(moduleName) {
```

Comment:
What's the purpose of this?

Reply:
@dfcowell as explained above we need to load the dependencies here so they will be accessible to renderer process.

Reply:
@dfcowell It might not need to bound to window but previously it was so I just kept like that. But I quite sure there is no way to just not load the required modules on the preload and handle that over to the renderer process when need with the switch I coded there. Cheers

Reply:
@dfcowell btw, require on rendering process works fine when on `yarn dev` mode but not when building the app with electron builder, `yarn dist` then `open dist/mac/SelfKey\ Identity\ Wallet.app/`. Just so you make sure to build it when testing any change. cheers

Reply:
@dfcowell it is not only us that have this issue https://stackoverflow.com/questions/49705639/configuring-electron-webpack-renderer-to-work-with-nodeintegration-false

Cheers